bricks/basic-bricks/src/grid-layout/index.ts

```diff
@@ -241,6 +259,14 @@ export class GridLayoutElement extends UpdatingElement {
 
       this.style.gridColumn = columnSpan === 1 ? "" : `span ${columnSpan}`;
       this.style.gridRow = rowSpan === 1 ? "" : `span ${rowSpan}`;
+      if (this.showGridBorder) {
+        document.documentElement.style.setProperty(
```

Comment:
这个属性设置在 this 上是否更好？否则两个 grid-layout 构件可能互相覆盖了这个属性。

```suggestion
        this.style.setProperty(
```